.flutter-plugins

```diff
@@ -1,8 +1,8 @@
 # This is a generated file; do not edit or check into version control.
-flutter_webrtc=/Users/utopia/development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_webrtc-0.9.16/
-path_provider=/Users/utopia/development/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-2.0.11/
-path_provider_android=/Users/utopia/development/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_android-2.0.22/
-path_provider_ios=/Users/utopia/development/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_ios-2.0.11/
-path_provider_linux=/Users/utopia/development/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_linux-2.1.7/
-path_provider_macos=/Users/utopia/development/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_macos-2.0.6/
-path_provider_windows=/Users/utopia/development/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider_windows-2.1.3/
+flutter_webrtc=/Users/vuphan/Development/rsupport/unity-view/flutter_janus_client/flutter-webrtc/
+path_provider=/Users/vuphan/.pub-cache/hosted/pub.dartlang.org/path_provider-2.0.11/
+path_provider_android=/Users/vuphan/.pub-cache/hosted/pub.dartlang.org/path_provider_android-2.0.22/
+path_provider_ios=/Users/vuphan/.pub-cache/hosted/pub.dartlang.org/path_provider_ios-2.0.11/
+path_provider_linux=/Users/vuphan/.pub-cache/hosted/pub.dartlang.org/path_provider_linux-2.1.7/
+path_provider_macos=/Users/vuphan/.pub-cache/hosted/pub.dartlang.org/path_provider_macos-2.0.6/
+path_provider_windows=/Users/vuphan/.pub-cache/hosted/pub.dartlang.org/path_provider_windows-2.1.3/
```

Comment:
It looks like a cache file.
Is it ok to ignore this file from git?

Reply:
ok, bro. I think it's okay